conf/application.conf

```diff
@@ -78,7 +78,8 @@ microservice {
 
     customs-declarations {
       host = localhost
-      port = 6790
+     # port = 9820
```

Comment:
Revert this file. Stay with correct port. No comments in `application.conf`

Reply:
added comment for local testing to point to customs-declarations if needed